addons/web_editor/static/src/js/editor/odoo-editor/src/base_style.scss

```diff
@@ -8,6 +8,19 @@ li.oe-nested {
         padding: 0.5rem;
     }
 }
+$sizes: 'xs', 'sm', 'md', 'lg', 'xl', 'xxl';
+.o_text_columns > .row {
+    @each $size in $sizes {
+        @for $i from 1 through 12 {
+            & > .col-#{$size}-#{$i}:first-of-type {
+                padding-left: 0;
+            }
+            & > .col-#{$size}-#{$i}:last-of-type {
+                padding-right: 0;
+            }
+        }
+    }
+}
```

Comment:
Shouldn't we add a class on those first and last columns via the editor instead ? Just wondering, because that is in fact a lot of very specific css rules.

Reply:
It would certainly be possible, it's a matter of choice I guess. Would you prefer that? This is a robust solution to me (though it does generate a bunch of css so not ideal indeed) but I guess adding a class of the first and last column in `columnize` (still can't get over that pun) would work (the downside being that if things change with `columnize` or around it in the future - eg being able to remove a column with backspace or something - we have to make sure the classes remain correct).

Reply:
Ouch yeah you're right. We don't want to have to maintain these classes.

Reply:
I didn't find a way to group the generated css in one line (like `a, b, c, d, e { rule }`) so it generates one rule per iteration here, that I find kind of too bad.